tests/test_funds.py

```diff
@@ -7,27 +7,54 @@
 from flask import Flask
 from flask import request
 from tests.test_data import BREKKY_ROUND
-from tests.test_data import FUND_DATA
 from tests.test_data import HARRY_S_BREAKFAST_FUND
 from tests.test_data import ROUNDS_IN_HARRY_S_BREAKFAST_FUND
+from tests.test_data import TEST_RESPONSE_FUND_DATA
 
 
 def test_all_funds_endpoint(client: Flask):
     """
     GIVEN the flask test client running our api
     WHEN we execute a GET request on "/funds"
-    THEN we expect a response containing our
-    initial data.
+    THEN we expect a response length consistent
+    with all inital data (including test data).
     """
     host_url = request.host_url
     url = host_url + "funds"
     api_response = client.get(url)
 
     raw_data = api_response.data
-    response_data = literal_eval(raw_data.decode("utf-8"))
+    number_of_funds_in_store_response = len(
+        literal_eval(raw_data.decode("utf-8"))
+    )
+    number_of_pre_configured_funds_in_store = 1
+    expected_number_of_funds = (
+        len(TEST_RESPONSE_FUND_DATA) + number_of_pre_configured_funds_in_store
+    )
+    asserts.assert_equal(
+        number_of_funds_in_store_response,
+        expected_number_of_funds,
+        msg_fmt="/funds didnt return the expected response, {msg}",
+    )
+
 
-    expected_data = FUND_DATA
+def test_all_funds_endpoint_includes_fund_test_data(client: Flask):
+    """
+    GIVEN the flask test client running our api
+    WHEN we execute a GET request on "/funds"
+    THEN we expect a response including the
+    test data.
+    """
+    host_url = request.host_url
+    url = host_url + "funds"
+    api_response = client.get(url)
+
+    raw_data = api_response.data
```

Comment:
Could we could just use `api_response.json` and avoid the gnarly parsing later on?